centos-pypa-rust/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quay.io/pypa/manylinux2014_x86_64
+
+WORKDIR /usr/src/app
+
+# download the rustup-init binary and checksum file
+ADD https://static.rust-lang.org/rustup/dist/aarch64-unknown-linux-musl/rustup-init rustup-init
+ADD https://static.rust-lang.org/rustup/dist/aarch64-unknown-linux-musl/rustup-init.sha256 rustup-init.sha256
```

Comment:
The `FROM` is for an x86_64 based image, but the `ADD`s are for an aarch64 architecture.